pagefaultgames/pokerogue
# [QoL] Add setting to control volume of UI sound efffects

## What are the changes?
Adds an option to the settings menu that toggles the sound effects from the UI.

## Why am I doing these changes?
Relevant issue: https://github.com/pagefaultgames/pokerogue/issues/3164

## What did change?



Adds a new variable to the settings menu that controls the volume of sounds originating from menus. This mostly means the "select" sound, along with the sounds for opening/closing some menus.

### Screenshots/Videos


![image](https://github.com/user-attachments/assets/66317370-d56e-46e0-ab16-4cb91350b53c)


## How to test the changes?





Manually tested by enabling the setting and moving around the main menu, starter select menu, and playing through a few battles to ensure all UI sounds were disabled

## Checklist
